app/assets/stylesheets/_shame.scss

```diff
@@ -467,3 +467,13 @@ section progress {
 ul > li {
     transition: background-color 0.3s ease 0s;
 }
+
+.external-link:after {
```

Comment:
Can there be a non-shameful solution? :)

Reply:
yes, sure :)
i forgot to move it into the file with general styles. I guess u use _page_layout for these styles.